src/authorization/AclAuthorizer.ts

```diff
@@ -18,7 +18,7 @@ import { Authorizer, AuthorizerArgs } from './Authorizer';
  * `acl:mode`, `acl:agentClass`, `acl:agent`, `acl:default` and `acl:accessTo`.
  * Does not support `acl:agentGroup`, `acl:origin` and `acl:trustedApp` yet.
  */
-export class SimpleAclAuthorizer extends Authorizer {
+export class AclAuthorizer extends Authorizer {
```

Comment:
`AllowEverythingAuthorizer`?

Reply:
I'm assuming this comment is meant for the `BasicAuthorizer` and not this one.

I thought of some names like that but didn't like any, so I assumed you would just say which ones to use in the review if you didn't like Basic 😂

Reply:
Indeed. `WebAclAuthorizer`?